src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -105,12 +105,17 @@ static std::string const BlankString;
 
 using json = nlohmann::json;
 
-InputProcessor::InputProcessor() : idf_parser(std::make_unique<IdfParser>()), data(std::make_unique<DataStorage>())
+const json &InputProcessor::schema()
 {
-    auto const embeddedEpJSONSchema = EmbeddedEpJSONSchema::embeddedEpJSONSchema();
-    schema = json::from_cbor(embeddedEpJSONSchema);
+    // avoid re-parsing embedded JSON schema by making this into a static const singleton
+    // because it is const, we don't have to worry about threading issues for creation or access
+    static const auto json_schema = json::from_cbor(EmbeddedEpJSONSchema::embeddedEpJSONSchema());
+    return json_schema;
+}
```

Comment:
This seems very reasonable.  A fully constant schema object only needs to be constructed once per running instance of EnergyPlus.